GPU/GLES/GPU_GLES.cpp

```diff
@@ -299,8 +299,10 @@ void GPU_GLES::CheckGPUFeatures() {
 	if (gl_extensions.EXT_texture_filter_anisotropic)
 		features |= GPU_SUPPORTS_ANISOTROPY;
 
-	if (gl_extensions.GLES3 || gl_extensions.EXT_gpu_shader4
-		|| (!gl_extensions.IsGLES && gl_extensions.VersionGEThan(3, 1)/*GLSL 1.4*/))
+	bool canUseInstanceID = gl_extensions.EXT_draw_instanced || gl_extensions.ARB_draw_instanced;
```

Comment:
If a device is GLES3, it need not (probably won't?) have this extension AFAIK.  It's part of GLES3 core and it's not necessary to list core extensions.

Examples:
http://opengles.gpuinfo.org/gles_generatereport.php?reportID=850
http://opengles.gpuinfo.org/gles_generatereport.php?reportID=1398

-[Unknown]